tests/testthat/test-linter_tags.R

```diff
@@ -13,6 +13,17 @@ test_that("available_linters returns a data frame", {
   expect_type(avail[["tags"]][[1L]], "character")
 })
 
+test_that("available_tags returns a character vector", {
+  tags <- available_tags()
+  tags2 <- available_tags(c("lintr", "not-a-package"))
+  empty <- available_tags("not-a-package")
+
+  expect_type(tags, "character")
+  expect_identical(tags, tags2)
+  expect_length(empty, 0L)
+  expect_true(all(available_linters()$tags[[1L]] %in% tags))
```

Comment:
why just the first row?

Reply:
Just a smell test.
Since the implementation is so simple, I didn't feel the need to loop all rows. LMK if you want to see this added.

Reply:
unlist(db$tags) instead?

Reply:
That's a fancy way of testing `unlist(db$tags) == unlist(db$tags)` 🤷🏻

Reply:
but then the current test is just a fancy way of testing 

db$tags[[1]] %in% unlist(db$tags)

no?

Reply:
... which at least ensures that `db$tags[[1]]` isn't a list.
What bothers me with `unlist()` is that that's precisely the implementation, so we don't test anything at all about the aggregation.

But we can test for uniqueness, via `anyDuplicated()`.

Reply:
I see, but that wasn't communicated by the test as written. maybe unlist(resursive=FALSE) would make that clearer?

anyway current tests look good enough, no sense blocking the PR any further 

>